api/scpca_portal/test/test_utils.py

```diff
@@ -21,6 +21,15 @@ def test_return_true(self):
             self.assertTrue(utils.boolean_from_string(v))
 
 
+class TestStringFromList(TestCase):
+    def test_return_joined_string(self):
+        list = ["a", "b", "c"]
+        expected_result = "a;b;c"
+        actual_result = utils.string_from_list(list)
+
+        self.assertEqual(actual_result, expected_result)
+
```

Comment:
Add a test case: `test_ignores_nonlist` that asserts any non list value passed in is nooped

Reply:
See commit at 43b3d7ce9ab5b6b649a92ec2f369e912da60a403.